### PR TITLE
My portfolio: contact form

### PR DESCRIPTION
**Pull request by BB-Simon**
1. Add contact form
2. Validate with html5 validation
3. Submit the form to formspree service

### CODE REVIEW

**Review by ha-manel — COMMENTED** [review]
# STATUS: INVALID 🔴 
Hi @BB-Simon 
I'm sure you put a lot of effort into this project but this PR is invalid because you worked alone in a pair programming activity 😓 your partner needs to have some contribution in your project and vice versa, UNLESS you have the permission to work alone on this project from Microverse staff then please put the screenshot of the proof in this PR (an email for example) 👍  

Kindly fix the issue and submit the PR for review again 👍 
Best of luck!

------
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by BB-Simon** [comment]
> 

Hi @ha-manel ,
This task I and my partner @Rudy-Menekam worked on this together, I mean we were giving advice and instructions to each other and I worked on this task on the zoom call with @Rudy-Menekam, So would please give me more clarity, on what exactly I need to work, for your kind information my partner did submit her task, there was no such an issue like this

**Comment by ha-manel** [comment]
@BB-Simon Hi, I'm really glad to hear that you've been collaborating with your partner and learning from eachother. However, in a pair programming project both of you need to have commits in the project, so your partner needs to have commits in this pull request to prove you worked together and to avoid getting invalidated. I hope this explains it all, best of luck to you both 😄

File: index.html [hunk 1]
```diff
@@ -232,74 +230,91 @@ <h4 id="title-contact">
...
+                </p>
+                <form action="https://formspree.io/f/bbsimon" method="post" class="form-container">
+                    <input type="text" placeholder="Your name" minlength="3" maxlength="50" required>
+                    <input type="email" placeholder="Your email" required>
+                    <textarea name="message" id="message" cols="30" rows="10" minlength="5" maxlength="550" placeholder="Write your message here" required></textarea>
+                    <input type="submit" value="Get It Touch">
+                </form>
+                <span class="footer-indicator"></span>
+            </div>
+        </section>
```
Comment (mckent05) [inline index.html]: - Well done on the work done so far with your project 👏, however, there seems to be an error when I try submitting details in the form. Please make sure that you have signed up on the [formspree service](https://formspree.io/html) and ensure your form is properly integrated using the `form_id` generated 👍 👍 
Error:
tgtgtg

File: index.html [hunk 2]
```diff
@@ -232,74 +230,91 @@ <h4 id="title-contact">
...
+                    <h3>Frameworks</h3>
+                    <img src="/assets/chevron-right.png" alt="Frameworks">
+                </li>
+                <div class="skills-list-item-divider"></div>
+                <li class="skills-list-item">
+                    <h3>Skills</h3>
+                    <img src="/assets/chevron-right.png" alt="Skills">
+                </li>
+                <div class="skills-list-item-divider"></div>
+            </ul>
```
Comment (mckent05) [inline index.html]: **OPTIONAL** - The images in this section of your page are not displaying, a likely cause could be that you are not navigating into the `assests` directory properly. I suggest you navigate into the directory like this  ` ./assets/chevron-right.png`. Please fix this 🙏

File: index.html [hunk 3]
```diff
@@ -232,74 +230,91 @@ <h4 id="title-contact">
...
+
+        <!-- Contact me section -->
+        <section class="contact-me-section">
+            <div class="contact-me-section-container">
+                <h1>Contact me</h1>
+                <p>
+                    If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it
+                </p>
+                <form action="https://formspree.io/f/bbsimon" method="post" class="form-container">
+                    <input type="text" placeholder="Your name" minlength="3" maxlength="50" required>
```
Comment (mckent05) [inline index.html]: - According to the project requirements, name input has maxlength="30" attribute. You used 50, please fix this 🙏

File: index.html [hunk 4]
```diff
@@ -232,74 +230,91 @@ <h4 id="title-contact">
...
+                    <input type="email" placeholder="Your email" required>
+                    <textarea name="message" id="message" cols="30" rows="10" minlength="5" maxlength="550" placeholder="Write your message here" required></textarea>
```
Comment (mckent05) [inline index.html]: - According to the project requirements, message input has a `maxlength="500"` attribute. You have used 550, please fix this 👍 👍

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @BB-Simon,

# STATUS: CHANGES REQUIRED ♻️  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost there

## Highlights
- All linter checks are passing ✔️ 
- Your PR is professional 👍 
- You have used the right workflow for this milestone 👏 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by jerryowusu — APPROVED** [review]
Hi Team :wave: , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

chelsea-mason-approve


## Highlights
All linters are passing :heavy_check_mark: 
Neat and well organised code :heavy_check_mark: 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**